Today Extension/TodayExtensionCell.swift

```diff
@@ -0,0 +1,215 @@
+
+    // MARK: Log vars
+
+    var rowNum: Int?
+
+    // MARK: View vars
+
+    var departureLabel: UILabel // ex: 10:00 AM at Collegetown Crossing
+    var destinationLabel: UILabel // ex: To Baker Flagpole
+    var liveLabel: UILabel
+    var liveIndicatorView: LiveIndicator
+    var busIcon: BusIcon?
+
+    // MARK: Spacing vars
+
+    let leftMargin: CGFloat =  12.0
+    let verticalMargin: CGFloat = 20.0 // top & bottom margin
+    let rightMargin: CGFloat = 16.0
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        departureLabel = UILabel()
+        destinationLabel = UILabel()
+        liveLabel = UILabel()
+        liveIndicatorView = LiveIndicator(size: .small, color: .clear)
+
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        contentView.addSubview(departureLabel)
+        contentView.addSubview(destinationLabel)
+        contentView.addSubview(liveLabel)
+        contentView.addSubview(liveIndicatorView)
+        if let busIcon = busIcon {
+            contentView.addSubview(busIcon)
+        }
+    }
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        departureLabel.snp.makeConstraints { (make) in
+            make.top.equalTo(verticalMargin)
+            make.leading.equalTo(74.0)
+            make.trailing.lessThanOrEqualToSuperview().inset(rightMargin)
+            make.height.equalTo(departureLabel.intrinsicContentSize.height)
+        }
+
+        if let busIcon = busIcon {
+            busIcon.snp.makeConstraints { (make) in
+                make.top.equalTo(verticalMargin)
+                make.leading.equalTo(leftMargin)
+                make.height.equalTo(busIcon.intrinsicContentSize.height)
+                make.width.equalTo(busIcon.intrinsicContentSize.width)
+            }
+        }
+
+        destinationLabel.snp.makeConstraints { (make) in
+            make.top.equalTo(departureLabel.snp.bottom).offset(2)
+            make.leading.equalTo(departureLabel)
+            make.trailing.lessThanOrEqualToSuperview().inset(rightMargin)
+            make.height.equalTo(destinationLabel.intrinsicContentSize.height)
+        }
+
+        if showLiveElements {
+            liveLabel.snp.makeConstraints { (make) in
+                make.bottom.equalToSuperview().inset(verticalMargin)
+                make.leading.equalTo(departureLabel)
+                make.height.equalTo(liveLabel.intrinsicContentSize.height)
+                make.width.equalTo(liveLabel.intrinsicContentSize.width)
+            }
+
+            liveIndicatorView.snp.makeConstraints { (make) in
+                make.centerY.equalTo(liveLabel.snp.centerY)
+                make.leading.equalTo(liveLabel.snp.trailing).offset(8)
+                make.trailing.lessThanOrEqualTo(rightMargin)
+                make.width.equalTo(liveIndicatorView.intrinsicContentSize.width)
+            }
+        }
+    }
+
+    func setUpCell(route: Route?, destination: String) {
```

Comment:
These types of statements (`self.route = route`) usually are in an init function, it might make more sense to make a custom init and set those variables there.